packages/cli/src/commands/account/transfer.ts

```diff
@@ -0,0 +1,42 @@
+import Command from '../../root-command'
+import { flags } from '@oclif/command'
+import { IAccount } from '@mesg/api/lib/account-lcd'
+import { ICoin } from '@mesg/api/lib/transaction'
+
+export default class AccountTransfer extends Command {
+  static description = 'Transfer token to another address'
+
+  static flags = {
+    ...Command.flags,
+    account: flags.string({
+      description: 'Account to use to send the token'
+    }),
+  }
+
+  static args = [{
+    name: 'TO',
+    required: true
+  }, {
+    name: 'AMOUNT',
+    required: true
+  }]
+
+  async run(): Promise<IAccount> {
+    const { args, flags } = this.parse(AccountTransfer)
+    const { account, mnemonic } = await this.getAccount(flags.account)
+
+    this.spinner.start(`Sending ${args.AMOUNT}atto to ${args.TO}`)
+    const coins: ICoin[] = [{
+      amount: args.AMOUNT,
+      denom: 'atto'
+    }]
+    const tx = await this.lcd.createTransaction(
+      [this.lcd.account.transferMsg(account.address, args.TO, coins)],
+      account
+    )
+    const txResult = await this.lcd.broadcast(tx.signWithMnemonic(mnemonic))
+    const res = await this.lcd.account.get(account.address)
+    this.styledJSON(res)
```

Comment:
why displaying the balance of the from account?
let's just display an success message like:
```
`Transfer of ${args.AMOUNT}atto from ${account.address}  to ${args.TO} executed with success`
```